doc/changelog/11-infrastructure-and-dependencies/13617-make+use_dune_for_ocaml.rst

```diff
@@ -0,0 +1,9 @@
+- **Changed:**
+  Coq's OCaml parts and tools [coq-core] are now built using Dune.
+  The main user-facing change is that Dune >= 2.5 is now required to
```

Comment:
Probably worth adding "but Dune >= 2.8" is recommended.

Reply:
Done